CI/setup_cvmfs_lcg105.sh

```diff
@@ -0,0 +1,23 @@
+# setup appropriate LCG release via cvmfs
+
+if test -e /etc/centos-release && grep 'CentOS Linux release 7' /etc/centos-release; then
+  lcg_os=centos7
+elif test -e /etc/centos-release && grep 'CentOS Stream release 8' /etc/centos-release; then
+  lcg_os=centos8
+# not centos. Check for RHEL  
+elif test -e /etc/redhat-release && grep 'Linux release 9' /etc/redhat-release; then
+  lcg_os=el9
+else
+  echo "Unsupported system" 1>&2
+  return
+fi
+
+lcg_release=LCG_105
+lcg_compiler=gcc11-opt
+lcg_platform=x86_64-${lcg_os}-${lcg_compiler}
+lcg_view=/cvmfs/sft.cern.ch/lcg/views/${lcg_release}/${lcg_platform}
+
+source ${lcg_view}/setup.sh
+# extra variables required to build acts
+export DD4hep_DIR=${lcg_view}
```

Comment:
I wonder if we should rather set `CMAKE_PREFIX_PATH`. but not critical

Reply:
These lines were copied from previous LCG setup scripts. 
Personally, I like them being in the setup script to keep the cmake command as clean as possible, but I'm happy with whatever you want.